File: src/com/s16/engmyan/MainApp.java
package com.s16.engmyan;

import android.app.Application;

public class MainApp extends Application {
	
	public MainApp() {
		super();
	}
	
	@Override
	public void onCreate() {
		ExpansionManager.tryMountExpansion(this);
		super.onCreate();
    }
	
	@Override
	public void onTerminate() {
		ExpansionManager.unmountExpansion(this, true);
		super.onTerminate();
    }
}

File: app/src/main/java/com/s16/engmyan/MainApp.kt
package com.s16.engmyan

import android.app.Application
import androidx.preference.PreferenceManager
import com.s16.engmyan.utils.UIManager

class MainApp : Application() {

    override fun onCreate() {
        super.onCreate()

        val preferences = PreferenceManager.getDefaultSharedPreferences(this)
        UIManager.setTheme(preferences.getString(Constants.PREFS_THEMES, ""))
    }
}